### ISSUE TITLE
Issue: Notifications with highest importance don't make sound when phone is on silent mode

### ISSUE THREAD

**Issue by ShruAlekha** [rafaelsetragni/awesome_notifications#319]
Hello, Thanks for the awesome plugin. I tried running the example app on Android and everything works great when the phone is on default mode. When I turn on the silent mode, even the notifications with max importance don't make any sound. How do I override this so the notifications show up with sound like an alarm application.

**Comment by rafaelsetragni** [comment]
No, this is not possible. The operational system and the user has priority over every execution that you do. Even schedules could be delayed if the device is in some special state, such as battery save mode. This is an philosophy that both Android and iOS follows and i agree 100% with them.

**Comment by ShruAlekha** [comment]
However, if you see alarm applications, they do have this feature. Even if the phone is silent, I would still like my alarm to ring in the night.

**Comment by rafaelsetragni** [comment]
I've discovered a way to do it on Android and iOS, but both of then require special permissions and you gonna need to do a request to Apple to allow your app to use it.

**Comment by ShruAlekha** [comment]
Thanks so much. Really appreciate it.

**Comment by ShruAlekha** [comment]
@rafaelsetragni any updates on this?

**Comment by rafaelsetragni** [comment]
Ive tried to implement the critical alert feature to enable the notifications to overide do not disturb / silent mode, but is way more complex than ive expected, so i decided to finish it only after 0.7.0 version.

**Comment by lukiinhas1616** [comment]
Any news?

**Comment by rafaelsetragni** [comment]
Yes! The 0.7.0 beta is already available and is being very promissor! Join us on Discord to keep updated.

**Comment by bejaeger** [comment]
Hi all,
found this very old thread. Are there any news on this feature?
Also, I couldn't join Discord as the invite link seems to have expired.
Thanks!

**Comment by rafaelsetragni** [comment]
Hi! Sorry for taking so long to respond to your issue on GitHub. I was focused on releasing the new 0.7.0 version of `awesome_notifications` and `awesome_notifications_fcm` and i didn't had time enough to do both. But now I can answer all your questions.

So now i'm asking you to recreate this topic using the new issue template. There's a lot of missing informations that i need to understand your problem, and all the instructions are already in the new issue template. Also, remember to check beforehand if your issue was posted by another user.

So I will automatically close all previous issues so far.
Sorry for the inconvenience and i will be waiting for your new issue request.

Thank you so much for your support!
